### PR TITLE
Group by year in graph

### PR DESCRIPTION
**Pull request by ghost**
Graph `labels` allows to use `year` filter, like:
```
..
'labels': 'delivery_date|year'
..
```
which groups results by year.

### CODE REVIEW

File: src/ralph/dashboards/tests/test_parser.py [hunk 1]
```diff
@@ -111,3 +111,59 @@ def test_key_sort_sorts_records_descending_when_minus_present(self):
...
+        )
+        expected = DataCenterAssetFullFactory.create_batch(
+            1, delivery_date='2016-01-01',
+        )
+        self.a_2015 = DataCenterAssetFullFactory.create_batch(
+            3, delivery_date='2017-01-01',
+        )
+        graph = GraphFactory(
+            params=self._get_graph_params({
+                'labels': 'delivery_date|year',
```
Comment (mkurek) [inline src/ralph/dashboards/tests/test_parser.py]: could you make some test with label containing `__`?

File: src/ralph/dashboards/models.py [hunk 2]
```diff
@@ -40,6 +40,42 @@ class ChartType(Choices):
     pie_chart = _('Pie Chart').extra(renderer=PieChart)
 
 
+class FilteringLabel:
```
Comment (mkurek) [inline src/ralph/dashboards/models.py]: why filtering? It doesn't have anything to do with filtering (?)

File: src/ralph/dashboards/models.py [hunk 3]
```diff
@@ -40,6 +40,42 @@ class ChartType(Choices):
...
+    """
+    Adds grouping-by-year feature to query based on `label_filter`
```
Comment (mkurek) [inline src/ralph/dashboards/models.py]: label_group?

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/dashboards/tests/test_parser.py (hunk 1 above)
Comment (ghost) [reply src/ralph/dashboards/tests/test_parser.py]: done

**Review by mkurek — APPROVED** [review]

**Review by ar4s — APPROVED** [review]

**Pull request merged**